docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+version: "3"
+services:
+  db:
+    image: mysql
+    volumes:
+      - ../trillian/storage/mysql/storage.sql:/docker-entrypoint-initdb.d/storage.sql
+      #   - /usr/local/var/mysql/:/var/lib/mysql
+    restart: always
+    environment:
+      MYSQL_PASSWORD: zaphod
+      MYSQL_USER: test
+      MYSQL_DATABASE: test
+      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+
+  trillian-log:
+    depends_on:
+      - db
+    image: trillian_log_server:latest
+    build: 
+      context: ../trillian
+      dockerfile: server/trillian_log_server/Dockerfile
+    restart: always
+    ports:
+      - "8090:8090" # rpcs
+      - "8091:8091" # metrics
+    environment:
+      DB_HOST: db:3306
+      DB_DATABASE: test
+      DB_USER: test
+      DB_PASSWORD: zaphod 
+
+  trillian-log-signer:
+    depends_on:
+      - db
+    image: trillian_log_signer:latest
+    build: 
+      context: ../trillian
+      dockerfile: server/trillian_log_signer/Dockerfile
+    restart: always
+    ports:
+      - "8092:8091" # metrics
+    environment:
+      DB_HOST: db:3306
+      DB_DATABASE: test
+      DB_USER: test
+      DB_PASSWORD: zaphod 
+
+  trillian-map:
+    depends_on:
+      - db
+    image: trillian_map_server:latest
+    build: 
+      context: ../trillian
+      dockerfile: server/trillian_map_server/Dockerfile
+    restart: always
+    ports:
+      - "8093:8090" # rpcs
+      - "8094:8091" # metrics
+    environment:
+      DB_HOST: db:3306
+      DB_DATABASE: test
+      DB_USER: test
+      DB_PASSWORD: zaphod 
+
+  kt-server:
+    depends_on:
+      - db
+      - trillian-log
+      - trillian-map
+    build:
+      context: .
+      dockerfile: ./cmd/keytransparency-server/Dockerfile
+    image: keytransparency-server:latest
+    restart: always
+    ports:
+      - "8080:8080" # json & grpc
+    environment:
+      LOG_ID: 8879615737060686335
```

Comment:
Can you please add a comment that `LOG_ID` and `MAP_ID` values should be updated, here and below?